POKEMON_OF_THE_DAY/build/src/index.js

```diff
@@ -0,0 +1,79 @@
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
```

Comment:
I figured that if we commit the package-lock (to save the user who pulls the repo the trouble of running `npm install`) we might as well commit the build folder as well (to save the user who pulls the repo the trouble of running `npm run build`)

Reply:
committing the package lock doesn't make it so you don't have to run npm install, it just makes it so that when you do, you have the option to get exactly the same versions as the previous install. I have never seen build files stored in github in this way. I think it's just adding noise to the repository, particularly in a case like this where this is mainly going to be used for reference

Reply:
thats a really helpful explanation! 
I have 86'd the build folder.